site/pages/docs/ref/wb-delete-pii/wb-delete-pii-en.hbs

```diff
@@ -0,0 +1,102 @@
+---
+{
+	"title": "Delete-PII",
+	"language": "en",
+	"category": "Plugins",
+	"categoryfile": "plugins",
+	"description": "Delete Personal Identifiable Information (PII) before form submit.",
+	"altLangPrefix": "wb-delete-pii",
+	"dateModified": "2022-02-22"
+}
+---
+<span class="wb-prettify all-pre hide"></span>
+
+<div class="alert alert-warning">
+	<h2>Limited feature</h2>
+	<p>Compatible with regular forms or with forms using Postback plugin. This feature will <strong>not work</strong> with <a href="https://wet-boew.github.io/wet-boew/demos/formvalid/formvalid-en.html">Form validation</a> plugin. </p>
+</div>
+
+<section>
+	<h2>Purpose</h2>
+	<p>This component provides the ability to delete the Personal Identifiable Information (PII) from a form fields before submit.</p>
+</section>
+
+<section>
+	<h2>Use when</h2>
+	<ul>
+		<li>forms might contain Personal Identifiable Information (PII), this plugin will identify and delete this information before form submit.</li>
+		<li>can also be used with Postback plugin</li>
+	</ul>
+</section>
+
+<section>
+	<h2>Do not use when</h2>
+	<ul>
+		<li>a form is using Form validation plugin</li>
+	</ul>
+</section>
+
+<section>
+	<h2>Working example</h2>
+	<ul>
+		<li><a href="../../../demos/wb-delete-pii/wb-delete-pii-en.html">English example</a></li>
+		<li><a hreflang="fr" href="../../../demos/wb-delete-pii/wb-delete-pii-fr.html">French example</a></li>
+	</ul>
+</section>
+
+<section>
+	<h2>How to implement</h2>
+	<ol>
+		<li>In the opening form tag, add the following data attribute having the value <code>true</code> : <code>data-wb-delete-pii="true"</code> and the class <code>wb-delete-pii</code>
+			<pre><code>&lt;form class=&quot;<strong>wb-delete-pii</strong>&quot; <strong>data-wb-delete-pii=&quot;true&quot; </strong>&gt;&lt;/form&gt;</code></pre>
```

Comment:
Can we make automate that step?

Can the first input only add a flag to it's parent form, so it will help to prevent to do some code duplication.

Like something like this in your code, when "$elm" is the jQuery object of the input
```
$( $elm.get( 0 ).form ).on( "submit", function().....)
```

Reply:
Fixed!